tests/backend/sqlalchemy.py

```diff
@@ -0,0 +1,51 @@
+from contextlib import contextmanager
+from typing import Iterator
+
+import pytest
+from sqlalchemy import MetaData, create_engine
+from sqlalchemy.exc import OperationalError
+from sqlalchemy.orm import Session, as_declarative
+
+from event_sourcery_sqlalchemy import SQLAlchemyBackendFactory
+from event_sourcery_sqlalchemy.models import configure_models
+
+
+@as_declarative()
+class DeclarativeBase:
+    metadata: MetaData
+
+
+configure_models(DeclarativeBase)
```

Comment:
I think this won't last when we'll introduce customization in storage models. But for now it looks fine